### PR TITLE
Update updateAd of existing projects to their projectUpdate

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to Giveth/giveth-dapps-v2#250

@CarlosQ96 
Everything seems fine except the migration, I get this error:

```
QueryFailedError: syntax error at or near "Feb"
    at QueryFailedError.TypeORMError [as constructor] (/Users/renjer/Documents/src/giveth/impact-graph/src/error/TypeORMError.ts:7:9)
    at new QueryFailedError (/Users/renjer/Documents/src/giveth/impact-graph/src/error/QueryFailedError.ts:9:9)
    at PostgresQueryRunner.<anonymous> (/Users/renjer/Documents/src/giveth/impact-graph/src/driver/postgres/PostgresQueryRunner.ts:245:19)
    at step (/Users/renjer/Documents/src/giveth/impact-graph/node_modules/typeorm/node_modules/tslib/tslib.js:143:27)
    at Object.throw (/Users/renjer/Documents/src/giveth/impact-graph/node_modules/typeorm/node_modules/tslib/tslib.js:124:57)
    at rejected (/Users/renjer/Documents/src/giveth/impact-graph/node_modules/typeorm/node_modules/tslib/tslib.js:115:69)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  query: '\n' +
    '            UPDATE project \n' +
    '            SET "updatedAt" =Tue Feb 15 2022 19:22:42 GMT+0330 (Iran Standard Time)\n' +
    '            WHERE id=1\n' +
    '            ',
  parameters: undefined,
  driverError: error: syntax error at or near "Feb"
```
The correct time should be something like this `2022-03-17 11:50:24.760456` can you help me on that?
we may do this on one query but it's hotfix and I don't have SQL experience as you, I think you may have idea

### CODE REVIEW

**Comment by CarlosQ96** [comment]
It's trying to interpolate the Date object with "February blah blah" into the query I think you have to change the format first and put the value there not the object. In the test utils I did something similar. 
```javascript
new Date().toISOString().split('T')[0]
```
That would make the date object ''2022-03-17'" for example and postgresql understands that

**Comment by mohammadranjbarz** [comment]
> It's trying to interpolate the Date object with "February blah blah" into the query I think you have to change the format first and put the value there not the object. In the test utils I did something similar.
> 
> ```js
> new Date().toISOString().split('T')[0]
> ```
> 
> That would make the date object ''2022-03-17'" for example and postgresql understands that

Thanks @CarlosQ96  the moments worked as you said in DM
now the PR is ready for review

File: migration/1647514950889-fixUpdatetAtOfProjects.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    for (const project of projects) {
+      const projectUpdates = await queryRunner.query(
+        `
+            SELECT * FROM project_update
+            where "projectId"=${project.id}
+            ORDER BY "createdAt" DESC
+            `,
+      );
+      if (projectUpdates.length === 0) {
```
Comment (CarlosQ96) [inline migration/1647514950889-fixUpdatetAtOfProjects.ts:22]: Remember projects always have at least 1 projectUpdate which is the main. In theory this shoudn't happen.

**Comment by mohammadranjbarz** [comment]
@CarlosQ96 Thanks for your feedbacks, I made your requested changes and it's ready for review again

**Review by CarlosQ96 — APPROVED** [review]
Thanks man! LGTM! Good job

**Pull request merged**